README.md

```diff
@@ -25,18 +25,14 @@ Check out the [examples](examples) directory.
 
 1. Install the library
 
-        git clone https://github.com/jupyter-incubator/sparkmagic
-        cd sparkmagic
-        pip install -e .
+        pip install sparkmagic
 
-2. (Optional) Install the wrapper kernels
+2. (Optional) Install the wrapper kernels. Do `pip show sparkmagic` and it will show the path where `sparkmagic` is installed at. `cd` to that location and do:
 
-        jupyter-kernelspec install remotespark/kernels/sparkkernel
-        jupyter-kernelspec install remotespark/kernels/pysparkkernel
+        jupyter-kernelspec install kernels/sparkkernel
```

Comment:
When I `pip show sparkmagic` I get this:

    Metadata-Version: 2.0
    Name: sparkmagic
    Version: 0.0.1
    Summary: SparkMagic: Spark execution via Livy
    Home-page: https://github.com/jupyter-incubator/sparkmagic
    Author: Jupyter Development Team
    Author-email: jupyter@googlegroups.org
    Installer: pip
    License: BSD 3-clause
    Location: c:\users\ristew\appdata\local\continuum\anaconda2\lib\site-packages
    Requires: autovizwidget, requests, hdijupyterutils
    Classifiers: ...

If I `cd` to that location then I can't run those commands because there's no `kernels` directory in `site-packages`. You should change these to 

    jupyter-kernelspec install sparkmagic/kernels/sparkkernel
    jupyter-kernelspec install sparkmagic/kernels/pysparkkernel